apis/orgpolicy/v1alpha1/policy_types.go

```diff
@@ -84,12 +84,21 @@ type OrgPolicyPolicyStatus struct {
 // OrgPolicyPolicyObservedState is the state of the OrgPolicyPolicy resource as most recently observed in GCP.
 // +kcc:proto=google.cloud.orgpolicy.v2.Policy
 type OrgPolicyPolicyObservedState struct {
+	// Basic information about the Organization Policy.
+	// +kcc:proto:field=google.cloud.orgpolicy.v2.Policy.spec
+	Spec *PolicySpecObservedState `json:"spec,omitempty"`
```

Comment:
Could we call this PolicySpec in the future? Non blocker, thanks for fixing HEAD 🙏🏼

Reply:
Yeah I found this confusing as well. Maybe we don't follow the naming of proto field for this one.